src/System.Net.NetworkInformation/tests/FunctionalTests/IPGlobalPropertiesTest.cs

```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Sockets;
+using System.Net.Test.Common;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.NetworkInformation.Tests
+{
+    public class IPGlobalPropertiesTest
+    {
+        private readonly ITestOutputHelper _log;
+        public static readonly object[][] Loopbacks = new[]
+        {
+            new object[] { IPAddress.Loopback },
+            new object[] { IPAddress.IPv6Loopback },
+        };
+
+        public IPGlobalPropertiesTest()
+        {
+            _log = TestLogging.GetInstance();
+        }
+
+        [Fact]
+        public void IPGlobalProperties_AccessAllMethods_NoErrors()
+        {
+            IPGlobalProperties gp = IPGlobalProperties.GetIPGlobalProperties();
+
+            TcpConnectionInformation[] activeConnections = gp.GetActiveTcpConnections();
+            IPEndPoint[] tcpListeners = gp.GetActiveTcpListeners();
+            IPEndPoint[] udpListeners = gp.GetActiveUdpListeners();
+
+            IPGlobalStatistics v4IpStat = gp.GetIPv4GlobalStatistics();
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                // OSX does not provide IPv6  stats.
+                IPGlobalStatistics v6IpStat = gp.GetIPv6GlobalStatistics();
+            }
+
+            IcmpV4Statistics v4IcmpStat = gp.GetIcmpV4Statistics();
+            IcmpV6Statistics v6IcmpStat = gp.GetIcmpV6Statistics();
+            TcpStatistics v4TcpSTat = gp.GetTcpIPv4Statistics();
+            TcpStatistics v6TcpStat = gp.GetTcpIPv6Statistics();
+            UdpStatistics v4UdpStat = gp.GetUdpIPv4Statistics();
+            UdpStatistics v6UdpStat = gp.GetUdpIPv6Statistics();
+        }
+
+        [Theory]
+        [MemberData(nameof(Loopbacks))]
+        public void IPGlobalProperties_TcpListeners_Succeed(IPAddress address)
+        {
+            using (var server = new Socket(address.AddressFamily, SocketType.Stream, ProtocolType.Tcp))
+            {
+                server.Bind(new IPEndPoint(address, 0));
+                server.Listen(1);
+                _log.WriteLine("listening on {0}", server.LocalEndPoint);
```

Comment:
Nit: Use interpolated string format instead?
```c#
_log.WriteLine($"listening on {server.LocalEndPoint}");
```

Reply:
I was commented on before when using interpolated. Is there some advantage or is that just personal or team preference?

Reply:
In product src it's more questionable, unless it's in some debug-only or trace-only code, as it can lead to extra expenses.  But it's also often more readable/maintainable, so in test code it can be preferred.